libc-bottom-half/headers/public/__header_sys_ioctl.h

```diff
@@ -4,6 +4,8 @@
 #define FIONREAD 1
 #define FIONBIO 2
 
+#define TIOCGWINSZ 0x101
```

Comment:
using random constant can be risky in the future...

Reply:
Definitely. I did this since the `FIONREAD` and `FIONBIO` constants where already redeclared as 1 and 2, which seems random enough; however, I went for an extra `0x100` to keep `FIO` and `TIO` kind of distinct. The usual implementation of this constant takes into account things such as the size of the result struct, which I suppose is relevant when implementing the syscall in the kernel. For our usecase, we don't really need the additional data, so I thought a simple number would suffice. I can change it to something else if you feel it's inappropriate.